core/src/main/resources/hudson/model/Label/index.jelly

```diff
@@ -38,19 +38,40 @@ THE SOFTWARE.
       <j:forEach var="a" items="${it.allActions}">
         <st:include page="summary.jelly" from="${a}" optional="true" it="${a}" />
       </j:forEach>
-
       <div>
         <h2>${%Nodes}</h2>
-        <j:forEach var="n" items="${it.sortedNodes}">
-          <j:set var="c" value="${app.getComputer(n.nodeName)}"/>
-          <j:set var="url" value="${rootURL}/${c.url}"/>
-          <nobr>
-            <a href="${url}" class="jenkins-link--with-icon"><l:icon src="${c.iconClassName}" /></a>
-            <st:nbsp/>
-            <a href="${url}" class="model-link inside">${c.displayName}</a>
-          </nobr>
-          <wbr/>
-        </j:forEach>
+        <table id="nodes_${it.displayName}" class="jenkins-table sortable">
+            <thead>
+              <tr>
+                <th class="jenkins-table__cell--tight">S</th>
+                <th initialSortDir="down">${%Name}</th>
+                <th>${%Description}</th>
+                <th align="right" tooltip="Number of total executors that belong to this node">${%Configured executors}</th>
+                <th align="right" tooltip="Number of busy executors that are carrying out some work right now">${%Busy executors}</th>
+                <th align="right" tooltip="Number of idle executors that can start working immediately">${%Idle executors}</th>
```

Comment:
Please make tooltips localizable (needs a resource file if it doesn't exist yet, because these aren't suitable keys for translation).

Reply:
Please see https://www.jenkins.io/doc/developer/internationalization/i18n-jelly-views/

If you just copied this from somewhere, could you provide a reference where? Unlocalizable strings are a bug, at least in core.

Reply:
Not from jelly code. Copied from header of java functions.

Reply:
Ok I see. In that case I produce same issue here : https://github.com/jenkinsci/jenkins/pull/6409/files
will be fixed ASSAP

Reply:
Hm this will be tricky. I can translate it in English, German, Slovakian an partly in Czech language. But what about the Chinese? Shall Add the key there and keep it empty only update DE && EN files?

Reply:
Providing an initial key in the .properties file does open up for further translations in other languages. You're welcome to provide additional translations into languages you feel fine with, but we won't force languages on you you're not familiar with.

Reply:
New strings are regularly only available in English, so don't worry about other languages unless you feel like doing that.

I just recommend you use the `.properties` file instead of just wrapping the long strings you have in `${%…}` to prevent minor corrections of the English text to have to affect all other translations (it's even worse with external translations, like https://plugins.jenkins.io/localization-zh-cn/ ).

Example of that approach:
https://github.com/jenkinsci/jenkins/blob/677fbb2a81fa8e9a0bc37ad5eca56565612e1f2e/core/src/main/resources/hudson/PluginManager/installed.jelly#L61-L63
https://github.com/jenkinsci/jenkins/blob/677fbb2a81fa8e9a0bc37ad5eca56565612e1f2e/core/src/main/resources/hudson/PluginManager/installed.properties#L24-L26

Reply:
> Providing an initial key in the .properties file does open up for further translations in other languages. You're welcome to provide additional translations into languages you feel fine with, but we won't force languages on you you're not familiar with.

thx I see.
I am from Slovakia and live over 10 years in Austria (the country without kangaroos :-) ) So German is also no problem. And English, yeh I try my best but I thing here are enough reviewers to correct my crappy English.

Reply:
localization is done.
+ new default .properties file with all missing keywords (translations)
+ unified header 'Projects tied to ' with the /computer/<nodeName> view
+ DE translations
+ SK translations I skipped, it is really to Hard because of unicode characters

Reply:
🎉 

> SK translations I skipped, it is really to Hard because of unicode characters

Editors like IntelliJ can transparently translate your Unicode input into the escape sequences. But seriously, there's no need to do that for this PR, we regularly accept contributions into Jenkins that lack the corresponding localizations (we'd never be able to integrate a change otherwise!).

Reply:
There is also extension in vs-code with works really good. So I have added also Slovakian translations.